### DISCUSSION TITLE
Show streak count in checkmark widget

### DISCUSSION THREAD

**Discussion by soramis-eldon** [iSoron/uhabits#2055] [Feature Requests]
I would love to have a way of displaying the current streak of an activity inside the checkmark widget - it's extremely motivating to me.

Compare e.g. the current widgets with how Duolingo (above) does it.

![Screenshot_2024-10-07-21-46-53-005_com microsoft launcher-edit](https://github.com/user-attachments/assets/b6acf5ee-48aa-4450-b9d3-2aa87cd5622f)

It would be great if that would be an option in the settings to toggle, e.g.: "Show streak count in yes/no checkmark widget".

**Comment by twojnarowski** [comment]
Upvote, I'd say that would be a great widget to have!
